doc/sphinx/language/gallina-extensions.rst

```diff
@@ -701,6 +693,9 @@ terminating functions.
 Depending on the ``{…}`` annotation, different definition mechanisms are
```

Comment:
```suggestion
Depending on the :n:`@fixannot` annotation, different definition mechanisms are
```

Above, it is written: "term must be built as a". This should be changed to something like "The body of the recursive definition must be built as a".